github_rules/github_org_ip_allowlist.yml

```diff
@@ -0,0 +1,46 @@
+AnalysisType: rule
+Filename: github_org_ip_allowlist.py
+RuleID: GitHub.Org.IpAllowlist
+DisplayName: GitHub Org IP Allow List modified
+Enabled: true
+LogTypes:
+  - GitHub.Audit
+Tags:
+  - GitHub
+Severity: Low
```

Comment:
Wondering if disabling the allowlist or destroying entries would warrant an increased severity?

Reply:
Thinking about it I think they probably all deserve at least a medium since this could be used to circumvent controls or lock valid users out of the codebase. I'm going to bump to a medium